doc/whats_new/older_versions.rst

```diff
@@ -787,6 +787,8 @@ Changelog
 
 - :ref:`olivetti_faces` by `David Warde-Farley`_.
 
+- :func:`model_selection.model_selection.permutation_test_score` by `Alexandre Gramfort`_.
+
```

Comment:
@amueller 
How can I deal with modifying the changelog retrospectively? 🤔 
The original function has been added to `scikits.learn.cross_val.py` module. How do I need to refer to it now?

Reply:
Adding it to an old whatsnew is not usually something we do. You can do a "versionadded" if you want (thought it would be the earliest whatsnew in the code base for sure). We don't really guarantee completeness of the changelog.